packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,230 @@
+      scope.setTag('runtime.name', 'node');
+      scope.setTag('server_name', processEnv._AWS_XRAY_DAEMON_ADDRESS);
+      scope.setTag('url', `awslambda:///${this._awsContext.functionName}`);
+    };
+
+    /**
+     * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+     * @hidden
+     */
+    const setParameters = (): void => {
+      // setting parameters in scope which will be displayed as additional data in Sentry dashboard
+      hub.configureScope((scope: Scope) => {
+        setTags(scope);
+        // runtime
+        setAdditionalRuntimeData(scope);
+        // setting the lambda parameters
+        setAdditionalLambdaParameters(scope);
+        // setting the cloudwatch logs parameter
+        setCloudwatchLogsData(scope);
+        // setting the sys.argv parameter
+        scope.setExtra('sys.argv', process.argv);
+      });
+    };
+
+    // timeout warning buffer for timeout error
+    const timeoutWarningBuffer: number = 1500;
+
+    /** check timeout flag and checking if configured Time In Milliseconds is greater than timeout Warning Buffer */
+    if (this._timeoutWarning === true && configuredTimeInMilliseconds > timeoutWarningBuffer) {
+      const configuredTimeInSec = Math.floor(configuredTimeInMilliseconds / 1000);
+      const configuredTimeInMilli = configuredTimeInSec * 1000;
+
+      /**
+       * This function is invoked when there is timeout error
+       * Here, we make sure the error has been captured by Sentry Dashboard
+       * and then re-raise the exception
+       * @param configuredTime  - configured time in seconds
+       * @hidden
+       */
+      const timeOutError = (configuredTime: number): void => {
+        setTimeout(() => {
+          /**
+           * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+           */
+          setParameters();
+
+          const error = new Error(
+            `WARNING : Function is expected to get timed out. Configured timeout duration = ${configuredTimeInSec +
+              1} seconds.`,
+          );
+
+          /** capturing the exception and re-directing it to the Sentry Dashboard */
+          hub.captureException(error);
+          Sentry.flush(this._flushTime);
+        }, configuredTime);
+      };
+
+      this._awsContext.callbackWaitsForEmptyEventLoop = false;
+      timeOutError(configuredTimeInMilli);
+    }
+
+    /**
+     * unhandled and handled exception
+     * @param error  - holds the error captured in AWS Lambda Function
+     * @param id  - holds event id value
+     * @param callback  - callback function
+     */
+    rapidRuntime.prototype.postInvocationError = async function(
+      error: Error,
+      id: string,
+      callback: () => void,
+    ): Promise<void> {
+      /**
+       * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+       */
+      setParameters();
```

Comment:
Same comments as for `timeOutError`

Reply:
made this change in the new pull request.